ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -126,9 +132,24 @@ objects:
 
         # test specific code
         # don't let clients impact the global kubeconfig
-        mkdir -p /go/src/github.com/openshift/
-        cp -r /tmp/shared/openshift-azure /go/src/github.com/openshift/
-        cd /go/src/github.com/openshift/openshift-azure/
+        if [[ "${TEST_IMAGE}" == *"/bushslicer"* ]]; then
+          TEST_NODE_OPTS="etcd:master:node"
```

Comment:
Is it possible to handle all of these steps in a script in the bushslicer repo and then call that script using `TEST_COMMAND`?

Reply:
ok, this is one of those things we need to consider very well. 
1. To move to bushslicer repo means - one more repo to interact with, where we dont have much power. So any time we need to merge anything or change anything - we will have to get approved, etc. It might be right place, but don't think it worth the pain.
2. We can move this to openshift-azure repo and move scripts as part of `/tmp/shared` with right target calls... 
3. We can host scripts somewhere outside and pull them in (not the biggest fan of this).
4. We keep them at a minimum in the template, as there is a small change we will be able to consolidate on "no templates at all" any time soon.

@kargakis lets discuss this in the morning :)

Reply:
Strongly do not prefer changes to openshift/release -- the jobs here should be as static as possible.